README.md

```diff
@@ -40,23 +40,38 @@ What is included in the software unit-test? What is not? Fill this table.
 |---------------------------|---------------|---
 Battery Data-accuracy       | No            | We do not test the accuracy of data
 Computation of maximum      | Yes           | This is part of the software being developed
-Off-the-shelf PDF converter | _enter Yes/No | _enter reasoning
-Counting the breaches       | _enter Yes/No | _enter reasoning
-Detecting trends            | _enter Yes/No | _enter reasoning
-Notification utility        | _enter Yes/No | _enter reasoning
+Computation of minimum      | Yes           | This is part of the software being developed
+Off-the-shelf PDF converter | No 			| We do not test the PDF converter
+Counting the breaches       | Yes	 		| This is part of the software being developed
+Detecting trends            | Yes			| This is part of the software being developed
+Notification utility        | No			| We do not test the End mail notification
 
 ### List the Test Cases
 
 Write tests in the form of `<expected output or action>` from `<input>` / when `<event>`
 
 Add to these tests:
 
-1. Write minimum and maximum to the PDF from a csv containing positive and negative readings
-1. Write "Invalid input" to the PDF when the csv doesn't contain expected data
-1. _enter a test
-1. _enter a test
+Positive Flow Test Cases:
+
+1. Write minimum and maximum to the PDF from a csv containing positive and negative readings.
```

Comment:
Good Classification of positive and negative test cases

Reply:
Thanks